application/common/models/Mfa.php

```diff
@@ -80,6 +82,26 @@ public function loadData(string $rpOrigin = '')
         }
     }
 
+    /**
+     * Adds ['id' => MfaWebauthn->id, 'label' => MfaWebauthn->label] to the Mfa's data property
+     *  for each of a webauthn type MFA's MfaWebauthn children
+     */
+    public function loadMfaWebauthns()
+    {
+        if ($this->type != self::TYPE_WEBAUTHN) {
+            return;
+        }
+
+        if (! isset($this->data)) {
+            $this->data = [];
+        }
+
+        $webauthns = MfaWebauthn::findAll(['mfa_id' => $this-> id]);
+        foreach ($webauthns as $webauthn) {
+            $this->data += ['id' => $webauthn->id, 'label' => $webauthn->label];
```

Comment:
I am not sure what this _should_ do, but it only keeps the first pair of `id` and `label` elements.

Reply:
OK, I saw something similar with a `+=`, but I guess it does something different than what I was expecting.

How about 
`
            $this->data[] = ['id' => $webauthn->id, 'label' => $webauthn->label];`